source/baikal/index.md

```diff
@@ -9,11 +9,8 @@ layout: default
 Baïkal is a Cal and CardDAV server, based on [sabre/dav][1], that includes an
 administrative interface for easy management.
 
-For more information, read the main website at [baikal-server.com][2].
```

Comment:
Should we link to the new site instead?

Reply:
There is no real replacement. The new website is `sabre.io/baikal` but that link would then reference itself.